src/runtime_src/core/edge/user/aie/graph.cpp

```diff
@@ -1001,10 +952,10 @@ xrtGraphReadRTP(xrtGraphHandle ghdl, const char *port, char *buffer, size_t size
 }
 
 int
-xrtSyncBOAIE(xrtGraphHandle ghdl, unsigned int bo, const char *dmaID, enum xclBOSyncDirection dir, size_t size, size_t offset)
+xrtSyncBOAIE(xclDeviceHandle handle, xrtBufferHandle bohdl, const char *gmioName, enum xclBOSyncDirection dir, size_t size, size_t offset)
 {
   try {
-    api::xrtSyncBOAIE(ghdl, bo, dmaID, dir, size, offset);
+    api::xrtSyncBOAIE(handle, bohdl, gmioName, dir, size, offset);
```

Comment:
This is the C-API implementation receiving const char*, but I think api::xrtSyncBOAIE or maybe what it calls should use std::string.

Reply:
done